tests/test_global_commands.py

```diff
@@ -134,3 +137,83 @@ def test_cycle_value_invalid_arg(hlwm):
     # calling it a second time fails
     hlwm.call_xfail(command) \
         .expect_stderr('Invalid value for "clients.my_foo": ')
+
+
+def test_use_tag(hlwm):
```

Comment:
There is a similar test in test_tags.py. Should we remove it?

Reply:
I've merged the test cases into one the test case `test_use_tag` in `test_global_commands.py`

Reply:
Thanks!